### ISSUE TITLE
Support jack-knifing

### ISSUE THREAD

**Issue by jowagner** [Jbar-ry/ud_folder#2]
While I spotted a script that makes a 90-10 split of the training data, I don't see anything to move the position of the 10%, to train 10 models and to apply them to each section.

For tagging dev and test data, we would want to combine the 10 models. Easiest is voting. Better will be to dump the softmax layer, calculate the average activation over the 10 models and then take the argmax.

**Comment by jowagner** [comment]
On second though, token-by-token voting may be a bad idea for downstream tasks such as parsing. While the tagging accuracy should improve with voting, voting can result in less plausible sequences of tags.
